### DISCUSSION TITLE
Make the "gamepad" extension installed by default

### DISCUSSION THREAD

**Discussion by tristanbob** [4ian/GDevelop#4267] [Enhancement ideas (and their technical discussions)]
## Description

Gamepads are a common input source for games.  The gamepad extension is stable and useful in a large percentage of desktop games.

This is also a requirement if we want to add gamepad controls to "default controls" on first-class behaviors like Platformer and Top-Down.

## Solution suggested

Make the "gamepad" extension installed by default.

**Comment by 4ian** [comment]
@LuniMoon Tagging you there, we may want to think about how, design speaking, we could make some extensions easier to find even if they are not installed, but considered important or useful.
